engines/dreamweb/stubs.cpp

```diff
@@ -963,6 +965,14 @@ void DreamWebEngine::useTimedText() {
 
 	const uint8 *string = (const uint8 *)_timedTemp._string;
 	printDirect(string, _timedTemp._x, _timedTemp._y, 237, true);
+	const char *theText = (const char*)string;
+	if (lastText != theText) {
```

Comment:
Is this check needed when using the `QUEUE_NO_REPEAT` option below? I would expect the NO_REPEAT to skip it if it is the same as the last text.

Reply:
The reason is that for somehow `QUEUE_NO_REPEAT` doesn't take effect, and the sentences keeps repeating itself until the next line arrives. Everything is actually good with only `_lastText != theText`. Do we want to just remove `QUEUE_NO_REPEAT`?

Reply:
We discussed for a separate game, and indeed I forgot that `QUEUE_NO_REPEAT` prevent repeating the sentence currently being spoken, but as soon as it has finished, it can speak it again. So the additional check here is indeed needed.